lighthouse-core/gather/driver.js

```diff
@@ -477,11 +480,15 @@ class Driver {
     const waitForLoad = _options.waitForLoad || false;
     const disableJS = _options.disableJavaScript || false;
```

Comment:
while you're in here, do you want to get rid of `_options` and make `options` a default `{}`